pom.xml

```diff
@@ -102,6 +102,7 @@
         <strimzi-oauth.version>0.5.0</strimzi-oauth.version>
 
         <junit-json-params.version>5.5.1-r0</junit-json-params.version>
+        <test-containers.version>1.14.0</test-containers.version>
```

Comment:
I think this should be added to the dependencyManagement section as well.

Reply:
Added